hypershift-operator/controllers/nodepool/nodepool_controller_test.go

```diff
@@ -1031,16 +1372,20 @@ status: {}
 				Client: fake.NewClientBuilder().WithObjects(tc.tuningConfig...).Build(),
 			}
 
-			got, err := r.getTuningConfig(context.Background(), tc.nodePool)
+			td, pp, err := r.getTuningConfig(context.Background(), tc.nodePool)
```

Comment:
`getTuningConfig` gets the ConfigMaps referenced under `nodepool.Spec.tuningConfig` from hcp namespace, validates its `data` as either a `Tuned` or `PerformanceProfile` and returns the manifests.
As far as I understand those ConfigMaps are created by the "user" and referenced in the `NodePool` 

I do not see any token generation, but maybe I am missing some detail. Could you please elaborate? Thanks

Reply:
Thanks for the explanation. 
Yes, I agree with you, there are two updates. 
Right now I do not see a way to avoid one of those updates but if that is a performance problem we can create a follow up issue to handle it.